nussl/core/constants.py

```diff
@@ -95,3 +96,10 @@
     ('accompaniment', 'bass+drums+other'), 
     ('linear_mixture', 'vocals+bass+drums+other')
 ])
+
+##### augmentation.py
+# These values are found in the ffmpeg documentation in filters
+# that use the level_in argument, however, not all filters that
+# use this state these bounds in documentation with pytest.raises(ValueError):
```

Comment:
Why is there a stray `pytest.raises` here? I don't fully understand this comment.